src/server/game/Entities/Creature/Creature.h

```diff
@@ -266,7 +272,7 @@ class Creature : public Unit, public GridObject<Creature>, public MovableMapObje
     bool _IsTargetAcceptable(Unit const* target) const;
     [[nodiscard]] bool CanIgnoreFeignDeath() const { return (GetCreatureTemplate()->flags_extra & CREATURE_FLAG_EXTRA_IGNORE_FEIGN_DEATH) != 0; }
 
-    // pussywizard: updated at faction change, disable move in line of sight if actual faction is not hostile to anyone
+    // Updated at faction change, disable move in line of sight if actual faction is not hostile to anyone
```

Comment:
Please keep those, they are a good indicator of what came from sunwell, and what we may need to change or remove in the future